streams/src/test/java/org/apache/kafka/streams/state/internals/MeteredSessionStoreTest.java

```diff
@@ -472,11 +472,26 @@ public void shouldThrowNullPointerOnRemoveIfKeyIsNull() {
         assertThrows(NullPointerException.class, () -> store.remove(null));
     }
 
+    @Test
+    public void shouldThrowNullPointerOnPutIfWrappedKeyIsNull() {
+        assertThrows(NullPointerException.class, () -> store.put(new Windowed<>(null, new SessionWindow(0, 0)), "a"));
```

Comment:
This test remind me, that the `SessionWindow` what is wrapped should not be `null` either.

Reply:
Yeah, sorry, I forget to do that sometimes.